docs/quick-start.md

```diff
@@ -40,7 +40,7 @@ RDDs have _[actions](programming-guide.html#actions)_, which return values, and
 
 {% highlight scala %}
 scala> textFile.count() // Number of items in this RDD
-res0: Long = 126
+res0: Long = 99
```

Comment:
Because this will change if `README.md` happens to change, you may wish to just comment that the number may be different, and so we don't have to update it again